tilequeue/command.py

```diff
@@ -949,6 +953,41 @@ def tilequeue_enqueue_tiles_of_interest(cfg, peripherals):
     logger.info('%d tiles of interest processed' % n_toi)
 
 
+def tilequeue_consume_tile_traffic(cfg, peripherals):
+    logger = make_logger(cfg, 'consume_tile_traffic')
+    logger.info('Consuming tile traffic logs ...')
+    logger.info(cfg.tile_traffic_log_path)
+
+    iped_dated_coords = None
```

Comment:
addressed in 8e4e864

adopted your suggestion - `tile_log_records`, sounds definitely better but also less explicit of what it actually is.